packages/shell-api/src/explainable-cursor.ts

```diff
@@ -15,10 +15,13 @@ export default class ExplainableCursor extends Cursor {
     this.verbosity = verbosity;
   }
 
-  async toReplString(): Promise<any> {
+  async asPrintable(): Promise<any> {
```

Comment:
It's only used for `asShellResult` but it's also the only method that gets overridden in the class definitions. So an alternative, where we don't have `asPrintable` could be:

Have every class that overrides the printable method define `asShellResult`, so it would have:
```ts
async asShellResult(): Promise<ShellResult> {
  return {
      value: <custom code for asPrintable>,
      type: <name of class>
  }
}
```
so for DB:
```ts
class Database {
  async asShellResult(): Promise<ShellResult> {
    return {
        value: this.name,
        type: 'Database'
    }
  }
```
And I thought there was something weird about having a class include it's name in a method (as opposed to being generated) but I don't have strong feelings either way.